src/backends/pgfplots.jl

```diff
@@ -120,50 +120,50 @@ end
 function _pgfplots_get_linestyle!(kwargs, plt)
     ls = plt[:linestyle]
     if haskey(_pgfplots_linestyles, ls)
-        kwargs[:style] *= _pgfplots_linestyles[ls]*","
+        push!(kwargs[:style], _pgfplots_linestyles[ls])
     end
 
-    kwargs[:style] *= "line width = $(plt[:linewidth]) pt"*","
+    push!(kwargs[:style], "line width = $(plt[:linewidth]) pt")
 end
 
 
 function _pgfplots_get_marker!(kwargs, plt)
     # Control marker shape
     mark = plt[:markershape]
-    kwargs[:style] *= _pgfplots_markers[mark]
+    push!(kwargs[:style], _pgfplots_markers[mark])
 
     # Control marker size
-    kwargs[:style] *= "mark size = $(plt[:markersize]/2),"
+    push!(kwargs[:style], "mark size = $(plt[:markersize]/2)")
 
     # Control marker colors and alphas
     α = plt[:markeralpha] == nothing ? 1.0 : plt[:markeralpha]
-    kwargs[:style] *= "mark options = {color=$(_pgfplots_get_color(plt, :markerstrokecolor)),"
-    kwargs[:style] *= mark == :dtriangle ? "rotate=180," : ""
-    kwargs[:style] *= "fill=$(_pgfplots_get_color(plt, :markercolor)),"
-    kwargs[:style] *= "fill opacity = $α,"
+    push!(kwargs[:style], "mark options = {color=$(_pgfplots_get_color(plt, :markerstrokecolor))")
+    mark == :dtriangle && push!(kwargs[:style], "rotate=180")
+    push!(kwargs[:style], "fill=$(_pgfplots_get_color(plt, :markercolor))")
+    push!(kwargs[:style], "fill opacity = $α")
```

Comment:
I certainly could, I was just trying to keep the line width sane. What is a multiline?